src/test/java/org/json/junit/JSONObjectTest.java

```diff
@@ -620,10 +620,6 @@ public void jsonObjectByBean1() {
         assertTrue("expected h\be\tllo w\u1234orld!", "h\be\tllo w\u1234orld!".equals(jsonObject.query("/escapeStringKey")));
         assertTrue("expected 42", Integer.valueOf("42").equals(jsonObject.query("/intKey")));
         assertTrue("expected -23.45e7", Double.valueOf("-23.45e7").equals(jsonObject.query("/doubleKey")));
-        // sorry, mockito artifact
```

Comment:
If we aren't going to validate the Mockito artifacts here, then I see no reason to use Mockito for this test case at all. It adds no benefit if we remove the assertions against it.

Reply:
I understand these checks don't work against the new version of Mockito which is required for Java 17+, If you can please try to find a solution that will do similar validations.